modules/aggregation-layers/src/gpu-grid-layer/gpu-grid-cell-layer.js

```diff
@@ -39,7 +39,7 @@ const defaultProps = {
   elevationScale: {type: 'number', min: 0, value: 1},
   extruded: true,
   fp64: false,
-  pickable: false, // TODO: add picking support (read from aggregated texture)
+  pickable: true,
```

Comment:
Removed setting this prop here and at other places. I was trying disable pick method for this composite layer, but figured it will be handled by base `CompositeLayer` class.